aries_cloudagent/anoncreds/holder.py

```diff
@@ -259,6 +279,63 @@ async def store_credential(
 
         return credential_id
 
+    async def store_credential_w3c(
+        self,
+        credential_definition: dict,
+        credential_data: dict,
+        credential_request_metadata: dict,
+        credential_attr_mime_types: dict = None,
+        credential_id: str = None,
+        rev_reg_def: dict = None,
+    ) -> str:
+        """Store a credential in the wallet.
+
+        Args:
+            credential_definition: Credential definition for this credential
+            credential_data: Credential data generated by the issuer
+            credential_request_metadata: credential request metadata generated
+                by the issuer
+            credential_attr_mime_types: dict mapping attribute names to (optional)
+                MIME types to store as non-secret record, if specified
+            credential_id: optionally override the stored credential id
+            rev_reg_def: revocation registry definition in json
+
+        Returns:
+            the ID of the stored credential
+
+        """
+        try:
+            secret = await self.get_master_secret()
+            cred_w3c = W3cCredential.load(credential_data)
+            await asyncio.get_event_loop().run_in_executor(
```

Comment:
I'm a bit confused what this is doing? If it is processing a cred_w3c credential wouldn't it have, and use a return value?

Reply:
We receive a W3C credential, but then we have to convert to an Indy credential to store.  So, we have to process the W3C credential before we can convert to Indy format, but we don't store the W3C version of the credential explicitely.